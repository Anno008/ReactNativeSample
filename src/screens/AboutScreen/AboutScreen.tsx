import * as React from "react";
import { Text, View } from "react-native";

export const AboutScreen: React.StatelessComponent = (): JSX.Element => (
  <View>
    <Text>About screen</Text>
  </View>
);
