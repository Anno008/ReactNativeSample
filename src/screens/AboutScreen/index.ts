export { default } from "screens/AboutScreen/AboutScreen";
