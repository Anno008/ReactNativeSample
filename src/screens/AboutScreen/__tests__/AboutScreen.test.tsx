import { shallow, ShallowWrapper } from "enzyme";
import React from "react";
import AboutScreen from "screens/AboutScreen";

describe("AboutScreen test", () => {
  it("should render", () => {
    const wrapper: ShallowWrapper = shallow(<AboutScreen />);
    expect(wrapper).toMatchSnapshot();
  });
});
