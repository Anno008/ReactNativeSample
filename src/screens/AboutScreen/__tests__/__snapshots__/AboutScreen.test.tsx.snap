// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutScreen test should render 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": <Text>
        About screen
      </Text>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": "About screen",
      },
      "ref": null,
      "rendered": "About screen",
      "type": [Function],
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": <Text>
          About screen
        </Text>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": "About screen",
        },
        "ref": null,
        "rendered": "About screen",
        "type": [Function],
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
